stage4/script1.js

```diff
@@ -0,0 +1,223 @@
+const questionsArray = [];
+const player1 = {
+  name: "",
+  score: 0,
+  input: document.getElementById("scoreInput1"),
+  scoreDisplay: document.getElementById("score1"),
+};
+const player2 = {
+  name: "",
+  score: 0,
+  input: document.getElementById("scoreInput2"),
+  scoreDisplay: document.getElementById("score2"),
+};
+
+function randomizeOptions() {
+  const options = document.querySelectorAll('[name="options"]');
+  const optionsArray = Array.from(options);
+
+  optionsArray.sort(() => Math.random() - 0.5);
+
+  optionsArray.forEach((option, index) => {
+    option.value = `Random Option ${index + 1}`;
+  });
```

Comment:
Shouldn't we update the options in the html here to match the new placements of the options?